include/crocoddyl/multibody/actuations/floating-base.hpp

```diff
@@ -53,7 +54,7 @@ class ActuationModelFloatingBaseTpl : public ActuationModelAbstractTpl<_Scalar>
 
   virtual boost::shared_ptr<Data> createData() {
     boost::shared_ptr<Data> data = boost::allocate_shared<Data>(Eigen::aligned_allocator<Data>(), this);
-    data->dtau_du.diagonal(-6).fill((Scalar)1.);
+    data->dtau_du.diagonal(-nvFirstJoint).fill((Scalar)1.);
```

Comment:
Change this code to:
```cpp
std::size_t fb_nv = state->get_pinocchio()->joints[1].nv();
data->dtau_du.diagonal(-fb_nv).fill((Scalar)1.);
```

Reply:
I tried that line before but it's not working, just not to add the variable I was not sure how to recover the state information I think I should use the ```state_``` there? Can you help me with this doubt?

Reply:
Yes, that's correct.